atom/browser/native_window_mac.mm

```diff
@@ -277,7 +278,29 @@ - (NSRect)window:(NSWindow*)window
 
 @end
 
-@interface AtomNSWindow : EventDispatchingWindow {
+@interface AtomPreviewItem : NSObject <QLPreviewItem>
+
+@property (nonatomic, retain) NSURL *previewItemURL;
```

Comment:
Although the linter doesn't pick this up, I think we typically write this as ` NSURL* previewItemURL` instead.